runtime/astar/src/xcm_config.rs

```diff
@@ -279,7 +279,7 @@ impl xcm_executor::Config for XcmConfig {
     type HrmpNewChannelOpenRequestHandler = ();
     type HrmpChannelAcceptedHandler = ();
     type HrmpChannelClosingHandler = ();
-    type XcmRecorder = ();
+    type XcmRecorder = PolkadotXcm;
```

Comment:
Might be a good idea to expand the integration test you added - since it didn't catch that empty implementation was being used.

The test doesn't have to be super complex, but maybe just a scenario where a _non-default_ return value is expected.

Reply:
Good suggestion! Now it is tested ✅

Reply:
Now the `XcmRecorder` from the runtime `XcmConfig` is tested - [e593d3b](https://github.com/AstarNetwork/Astar/pull/1358/commits/e593d3b8bc599be7bdb40e3284f3e85a714a9b93)